src/dlg_filter.cpp

```diff
@@ -34,7 +34,7 @@
 #define MAX_GAIN           20
 
 #define MAX_LOG10_Q         1.0
-#define MIN_LOG10_Q        -1.0 
+#define MIN_LOG10_Q        -1.0
 
```

Comment:
Do you use the Atom editor?  I've just started using it and it's always making these sort remove-the-space of commits for me....

Reply:
Ahh - wait a minute - this might be the changes i made for the 700E PR :slightly_smiling_face: ?

Reply:
Yeah, that's probably the 700E stuff. I don't remember making that change. 👍